configure.py

```diff
@@ -1654,9 +1652,6 @@ def configure_zstd(build_dir, mode):
         f.write(textwrap.dedent('''\
             build build/{mode}/iotune: copy build/{mode}/seastar/apps/iotune/iotune
             ''').format(**locals()))
-        f.write('build build/{mode}/scylla-package.tar.gz: package build/{mode}/scylla build/{mode}/iotune build/SCYLLA-RELEASE-FILE build/SCYLLA-VERSION-FILE build/debian/debian | always\n'.format(**locals()))
-        f.write('  pool = submodule_pool\n')
-        f.write('  mode = {mode}\n'.format(**locals()))
```

Comment:
Why are you removing this? It's still nice to be able to create a package using a ninja target.

In fact we may want to make rpm/deb be a ninja target.